memory.go

```diff
@@ -45,6 +45,7 @@ type memoryThresholdEvent struct {
 
 // MemoryThresholdEvent returns a new memory threshold event to be used with RegisterMemoryEvent.
 // If swap is true, the event will be registered using memory.memsw.usage_in_bytes
+//MemoryThresholdEvent returns a MemoryEvent interface so they can all be registered via the memory controllers memoryEvent method
```

Comment:
Reading this more, I think the first line of the comment (for both) should be changed to say it returns a `MemoryEvent` interface and then a subsequent line can mention what the event actually does/is registered for ("This event will trigger on the threshold being reached etc etc")